publiq-ui/pub-button.stories.mdx

```diff
@@ -1,7 +1,7 @@
 import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
 import PubButton, { ButtonVariant } from './pub-button.vue';
 
-<Meta title="pub-button" component={PubButton} />
+<Meta title="Components/pub-button" component={PubButton} />
```

Comment:
Note that every component will need to be put in this group. Otherwise you get something like this:

![Screenshot 2020-09-08 at 18 00 13](https://user-images.githubusercontent.com/959026/92500264-38c9f180-f1fd-11ea-9603-8491453b3bd8.png)

Reply:
👍